source/less/blocks/main-container.less

```
@@ -0,0 +1,19 @@
+.main-container__image {
+  position: relative;
+  padding: 90px 19px 85px;
+  background-color: #000000;
+  background-image: url("../../img/background/sedona-mountains-mobile.jpg");
```

```suggestion
  background-image: url("../img/background/sedona-mountains-mobile.jpg");
```
Неплохо бы уже запомнить, как правильно записываются относительные пути.

---

Андрей мне пути сам VS code подсказывает как писать , у меня есть папка "less"  в ней папка  "block" я выхожу из папки "block" в папку "less" и когда я поднимаюсь еще на уровень выше тогда я вижу что есть другие папки (если не поднимаюсь то другие папки не вижу) и потом я уже иду в них, да я вижу у меня постоянно замечания с путями, но мне так VS code подсказывает (и подсвечивает папки , вот у меня и получается такой путь ((( )

---

VS code показывает относительные пути в соответствии с файловой структуры проекта, но он ничего не знает о том, что файлы препроцессора .less будут преобразованы в один файл style.css.

Браузер разбирает относительные пути относительно файла, в котором они написаны. В данном случае это файл style.less. У нас файловая структура сайта (**не проекта!**) такая:
```bash
.                 # корневой каталог сайта
├── fonts/        # каталог шрифтов
├── img/          # каталог растровых и векторных изображений
├── css/          # каталог файлов стилей
│    └──  styles.css
└── index.html    # файл разметки страницы
```
Чтобы браузер мог найти путь к файлу изображения, который указан в styles.css ему нужно перейти на один уровень вверх относительно файла styles.css и тогда он сможет найти каталог img, а если в относительном пути указать два перехода наверх `../../`, то мы окажемся за пределами корневого каталога сайта (что в реальности невозможно из-за настроек сервера) и указанный путь будет неверным.

---

нам сказали используйте файл Less , я от него и отталкиваюсь, вот в этом и ошибка, я думаю что стили будут браться и этих файлов и от них отталкиваюсь(((